protocol/packages/swap/src/astroport/mod.rs

```diff
@@ -113,6 +91,24 @@ where
     }
 }
 
+trait Router {
+    const ADDRESS: &'static str;
+}
+
+#[cfg(feature = "dex-astroport_test")]
+impl Router for Impl {
+    // Source: https://github.com/astroport-fi/astroport-changelog/blob/main/neutron/pion-1/core_testnet.json
+    const ADDRESS: &'static str =
+        "neutron12jm24l9lr9cupufqjuxpdjnnweana4h66tsx5cl800mke26td26sq7m05p";
+}
+
+#[cfg(any(feature = "dex-astroport_main", feature = "dex-test_impl"))]
```

Comment:
we avoid `cfg` in the not-top-level modules hence we want the `lib.rs` to select the appropriate implementation